egs2/TEMPLATE/svs1/svs.sh

```diff
@@ -710,19 +711,19 @@ if ! "${skip_train}"; then
                 
                 _opts+="--train_shape_file ${svs_stats_dir}/train/text_shape.${token_type} "
                 _opts+="--train_shape_file ${svs_stats_dir}/train/singing_shape "
-                _opts+="--train_shape_file ${svs_stats_dir}/train/durations_shape "
-                _opts+="--train_shape_file ${svs_stats_dir}/train/score_shape "
+                # _opts+="--train_shape_file ${svs_stats_dir}/train/durations_shape "
+                # _opts+="--train_shape_file ${svs_stats_dir}/train/score_shape "
```

Comment:
Then it will cause error
```FileNotFoundError: [Errno 2] No such file or directory: 'exp/22k/svs_stats_raw_phn_None_zh/train/durations_shape'``` and also no score_shape

Reply:
I see. This may indicate a bug in the stats collection. @A-Quarter-Mile Do you have similar issues here?

Reply:
Let's just remove them for now (we are not using it anyway, if needed, we can easily add them back)

Reply:
Should we add if condition since it could be used for other svs tasks?

Reply:
I don't think so, let's simply remove it